index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="icon" href="/images/panda_face.svg" type="image/svg" />
+    <link rel="stylesheet" href="index.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-content">
+        <div class="logo-container">
+          <img src="images/panda_face.svg" alt="logo" class="logo" />
+          <a href="/"><h2>판다마켓</h2></a>
+        </div>
+        <a href="/login">
+          <button class="login-button">로그인</button>
+        </a>
+      </div>
+    </header>
+    <main>
+      <section class="banner">
+        <div class="banner-content">
+          <h1 class="title">
+            일상의 모든 물건을
+            <span class="change-line"> 거래해 보세요 </span>
+          </h1>
+          <a href="/items">
+            <button class="view-button">구경하러 가기</button>
+          </a>
+        </div>
+        <img
+          src="/images/home_top.svg"
+          alt="A panda mascot waving its hand"
+          class="banner-img"
+        />
+      </section>
+      <section class="home-cards">
+        <div class="home-card-container">
+          <div class="home-card-content">
+            <img
+              src="images/home_01.svg"
+              alt="Two pandas looking at a green shirt"
+              class="home-card-img"
+            />
+            <div class="home-card-info">
+              <div class="badge">Hot item</div>
+              <h1 class="title">
+                인기 상품을
+                <span class="change-line"> 확인해 보세요 </span>
+              </h1>
+              <p class="text">
+                가장 HOT한 중고거래 물품을
+                <br />
+
+                판다 마켓에서 확인해 보세요
+              </p>
```

Comment:
❗️ 수정요청
불필요한 공백은 없는 것이 좋습니다~

```suggestion
              <p class="text">
                가장 HOT한 중고거래 물품을
                <br />
                판다 마켓에서 확인해 보세요
              </p>
```